swift-recon.py

```diff
@@ -351,9 +356,19 @@ def main():
             maas_common.status_err('no ring provided to check')
         stats = swift_replication(args.ring)
     else:
-        maas_common.status_err(
-            'unrecognized command "{0}"'.format(args.recon)
-            )
+        raise CommandNotRecognized('unrecognized command "{0}"'.format(
```

Comment:
Just a thought but with argparse add_argument can take a choices keyword argument that will make sure args.recon is one of a sequence of values.